addons/calendar/models/calendar_event.py

```diff
@@ -893,10 +893,9 @@ def action_open_calendar_event(self):
     def action_sendmail(self):
         email = self.env.user.email
         if email:
-            for meeting in self:
-                meeting.attendee_ids._send_mail_to_attendees(
-                    self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
-                )
+            self.attendee_ids._send_mail_to_attendees(
```

Comment:
Do we really need to force send when using the "action _sendmail" method ? Can't it use the email queue ?